Source/Peripheral.swift

```diff
@@ -196,27 +207,31 @@ public class Peripheral {
 	 Immediately after that `.Complete` is emitted.
 	 */
 	public func discoverCharacteristics(identifiers: [CBUUID]?, service: Service) -> Observable<[Characteristic]> {
```

Comment:
This function is very similar to `discoverServices`. Maybe it's possible to extract common code for them?